packages/react-sdk/src/components/ImportWhiteboardDialog/ImportWhiteboardDialog.test.tsx

```diff
@@ -33,15 +42,22 @@ let widgetApi: MockedWidgetApi;
 
 afterEach(() => widgetApi.stop());
 
-beforeEach(() => (widgetApi = mockWidgetApi()));
+beforeEach(() => {
+  widgetApi = mockWidgetApi();
+});
 
 describe('<ImportWhiteboardDialog/>', () => {
   let Wrapper: ComponentType<PropsWithChildren<{}>>;
-  let whiteboardManager: jest.Mocked<WhiteboardManager>;
-  const onClose = jest.fn();
-  const onRetry = jest.fn();
+  let whiteboardManager: Mocked<WhiteboardManager>;
+  const onClose = vi.fn();
+  const onRetry = vi.fn();
   let data: WhiteboardDocumentExport;
 
+  afterEach(() => {
+    onClose.mockReset();
```

Comment:
Or create fresh `onClose` and `onRetry` in a `beforeEach`. If there should be a new one for each test run for me this would make more sense compared to clean up.